PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -39,9 +31,9 @@
         android:id="@+id/BoyfriendButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_above="@id/ContinueButtonMapGameBegins"
         android:layout_alignLeft="@id/HouseButton"
         android:layout_alignStart="@id/HouseButton"
+        android:layout_alignTop="@+id/SchoolButton"
```

Comment:
Thanks for highlighting that. You can also switch the order in which you declare the Boyfriend and the School button, so you don't need the `@+id`anymore. Since you are using a RelativeLayout, this won't change the behaviour